app/code/Magento/ConfigurableProduct/Block/Product/View/Type/Configurable.php

```diff
@@ -295,6 +295,7 @@ protected function getOptionPrices()
                 $tierPrices[] = [
                     'qty' => $this->localeFormat->getNumber($tierPrice['price_qty']),
                     'price' => $this->localeFormat->getNumber($tierPrice['price']->getValue()),
+                    'excl_tax_price' => (is_object($tierPrice['excl_tax_price'])) ? $this->localeFormat->getNumber($tierPrice['excl_tax_price']->getValue()) : $tierPrice['base_price'],
```

Comment:
I think it would be better to include `excl_tax_price` to result array only in case when it's not null.